lib/handlers.js

```diff
@@ -75,6 +75,7 @@ exports.put = function(req, res, next) {
 
   // Update in 1 atomic operation on the database if not specified otherwise
   if (this.shouldUseAtomicUpdate) {
+    this.update_options.new = true;
```

Comment:
This doesn't seem like expected behavior. I can't seem to find the docs for what exactly this does though. Can you point me in the right direction?